themes/bootstrap3/templates/RecordDriver/SolrDefault/data-summary.phtml

```diff
@@ -0,0 +1,13 @@
+<? 
+    $summaries = $this->driver->getSummary(); 
+    foreach ($summaries as $summary):
+        ?><?=$summary ?><br /><?
```

Comment:
I think we need $this->escapeHtml() around $summary here; VuFind generally assumes everything needs to be escaped.